.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Could we match the Gutenberg node required version? https://github.com/WordPress/gutenberg/blob/trunk/.nvmrc
```suggestion
20
```